dts/bindings/spi/spi-device.yaml

```diff
@@ -42,3 +42,10 @@ properties:
       enum:
         - 0
         - 32768
+    spi-cs-delay:
+       type: int
+       default: 0
+       required: false
```

Comment:
`required: false` can be omitted

Reply:
remove `default` as well.

Reply:
done.  Though I note most of the other properties in this file have `required: false`.